.golangci.yaml

```diff
@@ -21,12 +13,19 @@ issues:
         - staticcheck
       # https://staticcheck.io/docs/checks#SA4008 (The variable in the loop condition never changes, are you incrementing the wrong variable?)
       text: "SA4008:"
+  exclude-dirs:
+    - plugins/transport/dummy-alertmanager
+    - plugins/transport/dummy-events
+    - plugins/transport/dummy-metrics
+    - plugins/transport/dummy-logs
+    - plugins/application/print
+    - devenv
 
 linters:
   disable-all: true
   enable:
     - bodyclose
-    - depguard
+    # - depguard
```

Comment:
Did you look into replacements for any of these disabled linters? Sometimes there is a 1:1 replacement or a spiritual successor mentioned in the release notes.